optimade/filtertransformers/elasticsearch.py

```diff
@@ -109,16 +151,15 @@ def _query_op(self, quantity, op, value, nested=None):
             return Q(query_type, **{field: value})
 
         if op == "!=":
-            # != queries must also include an existence check
+            # != queries must also include an existence/v check
```

Comment:
What is the purpose of /v here ?
Is it a check mark that this point has been handled?
I usually use # TODO to mark comments about things that should still be implemented in the code.

Reply:
Looks like a typo :upside_down_face: 

```suggestion
            # != queries must also include an existence check
```